tasks/src/table-question-answering/data.ts

```diff
@@ -0,0 +1,68 @@
+import type { Task DataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{ 
+			description:"The Fewshot Table dataset consists of tables that naturally occur on the web, that are formatted as few-shot tasks for fine-tuning language models to improve their few-shot performance.",
+			id: "JeremyAlain/123_test",
+		},
+		{
+			description:"WikiSQL is a dataset of 80654 hand-annotated examples of questions and SQL queries distributed across 24241 tables from Wikipedia.",
+			id: "wikisql",
+		},
+	],
+	demo: {
+		inputs: [
+			{ table: [
+				["Rank", "Name", "No.of reigns", "Combined days"],
+				[1, 'lou Thesz', 3, 3749],
+                                [2, 'Ric Flair', 8, 3103],
+                                [3, 'Harley Race', 7, 1799],
```

Comment:
I think six lines are too long, we can have three lines, WDYT?
![Ekran Resmi 2022-10-14 12 29 04](https://user-images.githubusercontent.com/53175384/195825984-dabcd81c-baf7-4e56-97a6-41c11f068b02.png)